minmax_parameters.txt
Reserved 1 - THEXTRA                               !ROW 1
0.0000                                             !min
0.0000                                             !max
Reserved 2 - ICE_INDEX                             !ROW 2
0.0000                                             !min
0.0000                                             !max
Reserved 3 - GWSCALE                               !ROW 3
0.0000                                             !min
0.0000                                             !max
River roughness factor (WF_R2) (5 classes maximum) !ROW 4
0.0100                                             !min
0.7100                                             !max
WF_R2 - CLASS 2                                    !ROW 5
0.0100                                             !min
0.7100                                             !max
WF_R2 - CLASS 3                                    !ROW 6
0.0100                                             !min
0.7100                                             !max
WF_R2 - CLASS 4                                    !ROW 7
0.0100                                             !min
0.7100                                             !max
WF_R2 - CLASS 5                                    !ROW 8
0.0100                                             !min
0.7100                                             !max
Reserved 4 - Precipitation adjustment              !ROW 9
0.0000                                             !min precip_rat - Amount of precipitation adjustment - Not Used
0.0000                                             !max
Reserved 5 - Temperature adjustment                !ROW 10
0.0000                                             !min temper_inc - Amount of temperature adjustment - Not Used
0.0000                                             !max
DRNROW - DRAINAGE INDEX, CALCULATED DRAINAGE IS MULTIPLIED BY THIS VALUE                      !ROW 11
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min DRNROW          
1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000   !max
SDEPROW - THE PERMEABLE DEPTH OF THE SOIL COLUMN                                              !ROW 12
1.0000 0.4000 0.4000 0.4000 0.4000 0.4000 0.4000 0.4000 0.4000   !min SDEPROW 
4.1000 4.1000 4.1000 4.1000 4.1000 4.1000 4.1000 4.1000 4.1000   !max
FAREROW - WHEN RUNNING A MOSAIC, THE FRACTIONAL AREA THAT THIS TILE REPRESENTS IN A GRID CELL !ROW 13
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min FAREROW  
5.0000 5.0000 5.0000 5.0000 5.0000 5.0000 5.0000 5.0000 5.0000   !max
DDENROW - THE DRAINAGE DENSITY OF THE GRU IN KM/KM2                                           !ROW 14
0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001   !min DDENROW 
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
XSLPROW - AVERAGE OVERLAND SLOPE OF A GIVEN GRU                                               !ROW 15
0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010   !min XSLPROW 
0.0600 0.0600 0.0600 0.0600 0.0600 0.0600 0.0600 0.0600 0.0600   !max
XDNROW - HORIZONTAL CONDUCTIVITY AT A DEPTH OF h0 DIVIDED BY HORIZONTAL CONDUCTIVITY AT SURFACE !ROW 16
0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010 0.0010   !min XDNROW 
0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000   !max
MANNROW - MANNING ROUGHNESS COEFFICIENT                                                       !ROW 17
0.0100 0.0100 0.0100 0.0100 0.0100 0.0100 0.0100 0.0100 0.0100   !min MANNROW 
0.7100 0.7100 0.7100 0.7100 0.7100 0.7100 0.7100 0.7100 0.7100   !max
KSROW - HORIZONTAL CONDUCTIVITY AT SURFACE                                                    !ROW 18
0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001 0.0001   !min KSROW 
0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000   !max
SANDROW - PERCENTAGES OF SAND CONTENT OF SOIL LAYER 1                                         !ROW 19
-5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000   !min % OF SAND not organic in soil layer 1
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CLAYROW - PERCENTAGES OF CLAY CONTENT OF SOIL LAYER 1                                         !ROW 20
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF CLAY not organic or sand in soil layer 1
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ORGMROW - PERCENTAGES OF ORGANIC MATTER OF SOIL LAYER 1                                       !ROW 21
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF ORGANIC in soil layer 1
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
SANDROW - PERCENTAGES OF SAND CONTENT OF SOIL LAYER 2                                         !ROW 22
-5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000   !min % OF SAND not organic in soil layer 2
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CLAYROW - PERCENTAGES OF CLAY CONTENT OF SOIL LAYER 2                                         !ROW 23
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF CLAY not organic or sand in soil layer 2
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ORGMROW - PERCENTAGES OF ORGANIC MATTER OF SOIL LAYER 2                                       !ROW 24
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF ORGANIC in soil layer 2
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
SANDROW - PERCENTAGES OF SAND CONTENT OF SOIL LAYER 3                                         !ROW 25
-5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000 -5.000   !min % OF SAND not organic in soil layer 3
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CLAYROW - PERCENTAGES OF CLAY CONTENT OF SOIL LAYER 3                                         !ROW 26
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF CLAY not organic or sand in soil layer 3
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ORGMROW - PERCENTAGES OF ORGANIC MATTER OF SOIL LAYER 3                                       !ROW 27
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min % OF ORGANIC in soil layer 3
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ZSNLROW - LIMITING SNOW DEPTH BELOW WHICH COVERAGE IS < 100%                                  !ROW 28
0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000   !min ZSNLROW    **From MESH_parameters_hydrology.ini
1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000   !max
ZPLSROW - MAXIMUM WATER PONDING DEPTH FOR SNOW-COVERED AREAS                                  !ROW 29
0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000   !min ZPLSROW 
1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000   !max
ZPLGROW - MAXIMUM WATER PONDING DEPTH FOR SNOW-FREE AREAS                                     !ROW 30
0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000 0.1000   !min ZPLGROW 
1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000 1.0000   !max
LNZ0ROW - NATURAL LOGARITHM OF THE ROUGHNESS LENGTH FOR LAND COVER CATEGORY 1                 !ROW 31
-10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00   !min LNZ0ROW1   Column 1 **Atmospheric parameters from MESH_parameters_CLASS.INI 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALVCROW - VISIBLE ALBEDO FOR LAND COVER CATEGORY 1                                            !ROW 32
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALVCROW1  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALICROW - NEAR INFRARED ALBEDO FOR LAND COVER CATEGORY 1                                      !ROW 33
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALICROW1       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
RSMNROW - MINIMUM STOMATAL RESISTANCE FOR THE VEGETATION TYPE 1                               !ROW 34
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min RSMNROW1       
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT - COMMON VALUE 0.5 !ROW 35
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDAROW1 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION -OMMON VALUE 100 !ROW 36
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGAROW1  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LNZ0ROW - NATURAL LOGARITHM OF THE ROUGHNESS LENGTH FOR LAND COVER CATEGORY 2                 !ROW 37
-10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00   !min LNZ0ROW2   Column 2 **Atmospheric parameters from MESH_parameters_CLASS.INI 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALVCROW - VISIBLE ALBEDO FOR LAND COVER CATEGORY 2                                            !ROW 38
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALVCROW2  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALICROW - NEAR INFRARED ALBEDO FOR LAND COVER CATEGORY 2                                      !ROW 39
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALICROW2       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
RSMNROW - MINIMUM STOMATAL RESISTANCE FOR THE VEGETATION TYPE 2                               !ROW 40
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min RSMNROW2       
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT - COMMON VALUE 0.5 !ROW 41
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDAROW2 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION -OMMON VALUE 100 !ROW 42
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGAROW2  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LNZ0ROW - NATURAL LOGARITHM OF THE ROUGHNESS LENGTH FOR LAND COVER CATEGORY 3                 !ROW 43
-10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00   !min LNZ0ROW3   Column 3 **Atmospheric parameters from MESH_parameters_CLASS.INI 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALVCROW - VISIBLE ALBEDO FOR LAND COVER CATEGORY 3                                            !ROW 44
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALVCROW3  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALICROW - NEAR INFRARED ALBEDO FOR LAND COVER CATEGORY 3                                      !ROW 45
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALICROW3       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
RSMNROW - MINIMUM STOMATAL RESISTANCE FOR THE VEGETATION TYPE 3                               !ROW 46
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min RSMNROW3       
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT - COMMON VALUE 0.5 !ROW 47
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDAROW3 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION -OMMON VALUE 100 !ROW 48
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGAROW3  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LNZ0ROW - NATURAL LOGARITHM OF THE ROUGHNESS LENGTH FOR LAND COVER CATEGORY 4                 !ROW 49
-10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00   !min LNZ0ROW4   Column 4 **Atmospheric parameters from MESH_parameters_CLASS.INI 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALVCROW - VISIBLE ALBEDO FOR LAND COVER CATEGORY 4                                            !ROW 50
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALVCROW4  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALICROW - NEAR INFRARED ALBEDO FOR LAND COVER CATEGORY 4                                      !ROW 51
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALICROW4       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
RSMNROW - MINIMUM STOMATAL RESISTANCE FOR THE VEGETATION TYPE 4                               !ROW 52
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min RSMNROW4       
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT - COMMON VALUE 0.5 !ROW 53
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDAROW4 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGAROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION -OMMON VALUE 100 !ROW 54
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGAROW4  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LNZ0ROW - NATURAL LOGARITHM OF THE ROUGHNESS LENGTH FOR LAND COVER CATEGORY 5                 !ROW 55
-10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00 -10.00   !min LNZ0ROW5   Column 5 **Atmospheric parameters from MESH_parameters_CLASS.INI 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALVCROW - VISIBLE ALBEDO FOR LAND COVER CATEGORY 5                                            !ROW 56
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALVCROW5  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ALICROW - NEAR INFRARED ALBEDO FOR LAND COVER CATEGORY 5                                      !ROW 57
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ALICROW5       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
LAMXROW - MAXIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 1                                   !ROW 58
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMXROW1   Column 6 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
LAMNROW - MINIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 1                                   !ROW 59
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMNROW1  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CMASROW - ANNUAL MAXIMUM CANOPY MASS FOR VEGETATION TYPE 1 [kg m-2]                           !ROW 60
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min CMASROW1       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ROOTROW - ROOTING DEPTH FOR VEGETATION TYPE 1                                                 !ROW 61
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ROOTROW1       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
QA50ROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO LIGHT, COMMON VALUES - 30 TO 50 W/M2 !ROW 62
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min QA50ROW1 
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT, COMMON VALUES - 1.0 !ROW 63
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDBROW1  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION, COMMON VALUES - 5       !ROW 64
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGBROW1  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LAMXROW - MAXIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 2                                   !ROW 65
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMXROW2   Column 7 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
LAMNROW - MINIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 2                                   !ROW 66
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMNROW2  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CMASROW - ANNUAL MAXIMUM CANOPY MASS FOR VEGETATION TYPE 2 [kg m-2]                           !ROW 67
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min CMASROW2       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ROOTROW - ROOTING DEPTH FOR VEGETATION TYPE 2                                                 !ROW 68
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ROOTROW2       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
QA50ROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO LIGHT, COMMON VALUES - 30 TO 50 W/M2 !ROW 69
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min QA50ROW2 
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT, COMMON VALUES - 1.0 !ROW 70
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDBROW2  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION, COMMON VALUES - 5 !ROW 71
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGBROW2  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LAMXROW - MAXIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 3                                   !ROW 72
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMXROW3   Column 8 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
LAMNROW - MINIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 3                                   !ROW 73
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMNROW3  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CMASROW - ANNUAL MAXIMUM CANOPY MASS FOR VEGETATION TYPE 3 [kg m-2]                           !ROW 74
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min CMASROW3       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ROOTROW - ROOTING DEPTH FOR VEGETATION TYPE 3                                                 !ROW 75
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ROOTROW3       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
QA50ROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO LIGHT, COMMON VALUES - 30 TO 50 W/M2 !ROW 76
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min QA50ROW3 
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT, COMMON VALUES - 1.0 !ROW 77
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDBROW3  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION, COMMON VALUES - 5 !ROW 78
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGBROW3  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
LAMXROW - MAXIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 4                                   !ROW 79
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMXROW4   Column 9 
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
LAMNROW - MINIMUM LEAF AREA INDEX FOR THE VEGETATION TYPE 4                                   !ROW 80
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min LAMNROW4  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
CMASROW - ANNUAL MAXIMUM CANOPY MASS FOR VEGETATION TYPE 4 [kg m-2]                           !ROW 81
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min CMASROW4       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
ROOTROW - ROOTING DEPTH FOR VEGETATION TYPE 4                                                 !ROW 82
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min ROOTROW4       
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
QA50ROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO LIGHT, COMMON VALUES - 30 TO 50 W/M2 !ROW 83
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min QA50ROW4 
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max
VPDBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO VAPOR PRESSURE DEFICIT, COMMON VALUES - 1.0 !ROW 84
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min VPDBROW4  
100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00 100.00   !max
PSGBROW - COEFFICIENT GOVERNING THE RESPONSE OF STOMATES TO SOIL WATER SUCTION, COMMON VALUES - 5       !ROW 85
0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 0.0000   !min PSGBROW4  
1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0 1000.0   !max


Driver/MESH_Driver/FLAGS.F90
      MODULE FLAGS

!-      INTEGER HOURLYFLAG
      INTEGER RESUMEFLAG
      INTEGER SAVERESUMEFLAG 
      INTEGER SHDFILEFLAG
      INTEGER SOILINIFLAG
      INTEGER NRSOILAYEREADFLAG
!-      INTEGER CONFLAGS
      INTEGER RELFLG
!-      INTEGER OPTFLAGS
!-      INTEGER INTERPOLATIONFLAG
      INTEGER SUBBASINFLAG
      INTEGER R2COUTPUTFLAG
      INTEGER OBJFNFLAG
      INTEGER WINDOWSIZEFLAG
      INTEGER WINDOWSPACINGFLAG
      INTEGER METRICSSTATSOUTFLAG
      INTEGER METRICSFILTEROBSFLAG
      INTEGER METRICSSPINUP
      INTEGER METRICSINCLUDESPINUP
      INTEGER FROZENSOILINFILFLAG
!> JUN 03/11 - DGP      ADDED FOR WATDRN3
!> JUL 10/11 - DGP      WD3FLOW
!> AUG 19/11 - DGP      WD3BKFC
      INTEGER WD3
      INTEGER WD3NEWFILE
      INTEGER WD3FLOW
      INTEGER WD3BKFC
      INTEGER PBSMFLAG
!> JAN 30/14 - KCK     ADDED LOCATIONFLAG
      INTEGER LOCATIONFLAG

!> JUL 17/14 -GSA- GET FIELDS BALANCEs
      INTEGER OUTFIELDSFLAG

!> OCT 16/14 -GSA- GET Geothermal flux
      INTEGER GGEOFLAG
!> NOV 18/14 -GSA- Variable in/out file names
      INTEGER VARIABLEFILESFLAG

!-      INTEGER BASINBALANCEOUTFLAG
      integer :: BASINAVGEBFILEFLAG = 0
      integer :: BASINAVGEVPFILEFLAG = 0
      INTEGER MODELINFOOUTFLAG
      INTEGER :: BASINSWEOUTFLAG = 0
!> FEB 8/17 -GSA- Reservoir
      INTEGER RESERVOIRFLAG

      END
